### PR TITLE
Reduce ambiguity in spec

### PR DESCRIPTION
**Pull request by almarklein**
In prepping https://github.com/pygfx/wgpu-py/pull/202 I ran into some things which are somewhat ambiguous. This improves that.

### CODE REVIEW

**Comment by almarklein** [comment]
After this is merged I'll publish a release.

File: docs/examples/index.rst [hunk 1]
```diff
@@ -16,5 +16,6 @@ This is a list of example notebooks demonstrating the use of jupyter_rfb:
     pygfx2.ipynb
     pygfx3.ipynb
     video.ipynb
+    visibility_check.ipynb
```
Comment (djhoese) [inline docs/examples/index.rst:19]: Would it be worth it to do do `:glob:` at the top of the toctree and then do `*.ipynb`? I think that is supposed to work.

File: jupyter_rfb/events.py [hunk 2]
```diff
@@ -4,10 +4,9 @@
 Events are simple dict objects containing at least the key `event_type`.
 Additional keys provide more information regarding the event.
 
-Note that in jupyter_rfb, the *key_down*, *key_up*, and *wheel* events
-only apply when the widget has focus (having received a pointer down).
```
Comment (djhoese) [inline jupyter_rfb/events.py:8]: Is this information no longer provided? Should it be?

**Review by djhoese — APPROVED** [review]
I had a couple questions/suggestions, but otherwise looks good to me.

File: jupyter_rfb/events.py (hunk 2 above)
Comment (almarklein) [reply jupyter_rfb/events.py:8]: It has moved down to a separate subsection about focus.

File: docs/examples/index.rst (hunk 1 above)
Comment (almarklein) [reply docs/examples/index.rst]: Nice, it does! (except only with `*`, not with `*.ipynb`)

**Pull request merged**